index.bs

```diff
@@ -587,6 +587,57 @@ See also:
 * [[#secure-context]]
 * [[#consent]]
 
+<h3 id=text-formats>Make textual formats for people</h3>
+
+Design textual formats that can be easily produced and consumed by people.
+
+Favor readability over compactness. 
```

Comment:
```suggestion
Favor readability over compactness. 
Textual formats improve [transparency](https://www.w3.org/TR/ethical-web-principles/#transparent).
```

Reply:
This sentence reads a bit out of place here, though I have no concrete suggestion for where to move it :/

Reply:
Yeah, I struggled with that too.  Let me try again.